### PR TITLE
refactor(*): cleanup repo 🧹

### LINKED ISSUE #38: request: allow usage of attached images as base for meme CC
Allow the usage of Attached images as the base for the meme instead of their link. If a attachment is found the command will require only top text as arg.
Also make the Bottom Text optional, and if is not provided use **Bottom Text** as default ~~because this is a meme by itself~~

### LINKED ISSUE #194: Formatting trigger in code
i have seen several users copy the trigger with ` so maybe removing that or using a more elegant way mighz be better.
In this example i used the -be cc

### LINKED ISSUE #222: proposal: remove instructions in leading comments completely
Now that we have the website and associated infra set up (we have a job making sure the website builds and a job automatically deploying the website on new commits to `master`), I propose that instructions be removed from leading comments completely. All that would be left would be 1) a link to the associated page on the website and 2) information about the author of the CC.

**Motivation**
I think it goes without saying that the website is a far better place to browse and view custom commands than this GitHub repository. The question arises as to why exactly we need to keep instructions in leading comments anymore. Personally, I don't see any benefit of doing so, but I do see several downsides. In particular, if we want to keep the website up to date with the code here, we currently need to change up to 4 things in a single PR:
1) We need to change the actual code.
2) We need to change the instructions in the leading comment.
3) We need to change the code on the website. 
4) We need to change the instructions on the website.

Only 2 of those 4 things are really necessary, the rest is just repetitive work. If we remove the need to keep instructions in header comment, that's only 3 things to change at max, rather than 4. To me, that outweighs any benefit keeping instructions in leading comments could bring.

What do we think of this?

### PR DESCRIPTION
**Pull request by jo3-l**
gee well this is awkward i renamed the branch and it closed the PR upstream too
makes sense now but yeah

---

**Summary**

* Shorten leading comments. No more detailed instructions, just a short description + link to website page + author.
* Move custom command code to `src/`.
* Change custom command filenames to `snake_case`, to be consistent with Go.
* Don't duplicate code on the website. Instead, include the content from the file using a plugin.
* Rewrite pages on the website to make use of Docusaurus features like line highlighting, admonitions, and so on. Also use a consistent style across all pages.
* Probably some more things I've forgotten

**Progress**

* [x] Move custom command to `src/`
* [x] Change custom command filenames to `snake_case`
* [x] Replace <LINK> placeholders in leading comments with actual links
* [x] Rewrite website pages
  * [x] `afk/`
  * [x] `birthday/`
  * [x] `code-snippets/`
  * [x] `fun/`
  * [x] `birthday/`
  * [x] `info/`
  * [x] `leveling/`
  * [x] `moderation/`
  * [x] `suggestion/`
  * [x] `utilities/`
  * [x] `tags/`
* [x] Shorten leading comments
  * [x] `afk/`
  * [x] `birthday/`
  * [x] `code-snippets/`
  * [x] `fun/`
  * [x] `birthday/`
  * [x] `info/`
  * [x] `leveling/`
  * [x] `moderation/`
  * [x] `suggestion/`
  * [x] `utilities/`
  * [x] `tags/`

Closes #222 
Closes #194
Closes #38 (fix for this was included in this and I'm too lazy to move it out)

### CODE REVIEW

**Comment by jo3-l** [comment]
:D 2 sections to go and then website changes will be done

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -57,6 +57,6 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Check template syntax
-        uses: jo3-l/action-check-yag-tmpl-syntax@v1.0.1
+        uses: jo3-l/action-check-yag-tmpl-syntax@v2
         with:
-          include: '**/*.go.tmpl'
+          include: '**/**.go.tmpl'
```
Comment (jo3-l) [inline .github/workflows/continuous-integration.yml:62]: reminder to self: revert this even though both work

File: website/docs/fun/qotd.md [hunk 2]
```diff
@@ -1,84 +1,46 @@
...
 
-**Setup:**
+This command helps you manage a "Question of the Day" channel. It will resend QOTD messages and sends automatic reports about the last day's QOTD to a channel of your choice once you create a new QOTD.
 
-- Set the restrictions to only run in your QOTD channel
-
-**Trigger Type:** `Regex`
+## Trigger
 
+**Type:** `Regex`<br />
```
Comment (l-zeuch) [inline website/docs/fun/qotd.md:10]:
```suggestion
**Type:** `Regex`<br>
```
wtf_dude

File: website/docs/fun/repeat.md [hunk 3]
```diff
@@ -1,44 +1,26 @@
...
 title: Repeat Phrase
 ---
 
-This command repeats a given phrase.
+This command repeats text provided.
 
-**Trigger Type:** `Command`
+## Trigger
 
+**Type:** `Command`<br />
```
Comment (l-zeuch) [inline website/docs/fun/repeat.md:10]: please

File: website/docs/moderation/nickname-moderation.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (l-zeuch) [inline website/docs/moderation/nickname-moderation.mdx:1]: `*.mdx`????

File: website/docs/moderation/slowmode.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (l-zeuch) [inline website/docs/moderation/slowmode.mdx:1]: also `*.mdx`?

File: website/docs/suggestion/main-cc.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 2
+title: Main CC
+---
+
+Main suggestion command, provides CRUD functionality for suggestions and admin commands to manage suggestions.
```
Comment (l-zeuch) [inline website/docs/suggestion/main-cc.md:6]: maybe write it in whole words?
I had to google what `CRUD` even means

File: website/docs/tags/main-cc.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 2
+title: Main CC
+---
+
+Main tag command, provides CRUD tag functionality.
```
Comment (l-zeuch) [inline website/docs/tags/main-cc.md:6]: here same thing with `CRUD`

File: website/docs/utilities/edit.md [hunk 8]
```diff
@@ -1,164 +1,69 @@
...
+| `-url <url>`         | Sets the URL of the embed in the message             | `-url https://www.youtube.com/watch?v=oHg5SJYRHA0` |
+| `-author <name>`     | Sets the author name of the embed in the message     | `-author foo`                                      |
+| `-authoricon <url>`  | Sets the author icon of the embed in the message     | `-authoricon https://i.imgur.com/vfbFEif.jpeg`     |
+| `-footer <text>`     | Sets the footer text of the embed in the message     | `-footer bar baz`                                  |
+| `-footericon <url>`  | Sets the footer icon URL of the embed in the message | `-footericon https://i.imgur.com/vfbFEif.jpeg`     |
+| `-color <hex>`       | Sets the color of the embed in the message           | `-color FF0000`                                    |
+
+#### Flags to modify edit behaviour
+
+- `-force` - Completely overwrites the existing message. By default, the changes are merged into the existing message - for example, `-edit 123 -color FF0000` would just edit the color and keep the embed. However, by supplying the `-force` flag, `-edit 123 -color FF0000` would remove any other parts of the embed, keeping only the new color.
```
Comment (l-zeuch) [inline website/docs/utilities/edit.md:39]:
```suggestion
- `-force` - Completely overwrites the existing message. By default, the changes are merged into the existing message - for example, `-edit 123 -color FF0000` would just edit the color and keep the embed. However, by supplying the `-force` flag: `-edit 123 -color FF0000 -force` would remove any other parts of the embed, keeping only the new color.
```

**Review by l-zeuch — CHANGES_REQUESTED** [review]
This was a pain to review.
Please excuse the duplicate comments regarding the `<br>` tag, I simply wanted to make a point. According to the HTML standard, the correct way is `<br>` instead of `<br />` or whatever. I skipped a few files, but seeing as this is in the template, I suppose they're in every file which used that template.

Then there were also a few lacking newlines - please add a trailing newline to files where it's missing, as well, as by POSIX 3.206.

The rest should become evident through my comments/suggestions.
I am a bit confused over the `*.mdx` files - are those typos (seeing as there are only two of them) or are they intended extensions?

**Comment by jo3-l** [comment]
.mdx is an extension of Markdown that supports importing JavaScript components.
Certain commands use Wolf's permission checking system which supports some permissions, duplicating that every time was tiresome so I just wrote a component for it.

File: website/docs/moderation/slowmode.mdx (hunk 5 above)
Comment (jo3-l) [reply website/docs/moderation/slowmode.mdx:1]: see comment

File: website/docs/moderation/nickname-moderation.mdx (hunk 4 above)
Comment (jo3-l) [reply website/docs/moderation/nickname-moderation.mdx:1]: see comment

File: src/code_snippets/LICENSE.md [hunk 9]
```diff
@@ -23,4 +23,4 @@ OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
 ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
 
-For more information, please refer to http://unlicense.org/
+For more information, please refer to \<http://unlicense.org/>
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply src/code_snippets/LICENSE.md:26]: yes

[the remark this answers is not in the document]
Comment (l-zeuch) [reply src/code_snippets/LICENSE.md:26]: ok good

File: src/fun/uwuify.go.tmpl [hunk 10]
```diff
@@ -1,8 +1,8 @@
 {{/*
-	This command "uwuifies" text. The best way to see what the heck it does is to add it to your own server and test it.
-	Usage: `-uwuify <text>`
+	U-uwufies text~~
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply src/fun/uwuify.go.tmpl:2]: lgtm

File: website/TEMPLATE.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+sidebar_position: replace-this-with-a-number
+title: name-of-your-command
+---
+
+Detailed command description goes here.
+
+## Trigger
+
+**Type:** `trigger-type-here`<br />
+**Trigger:** `trigger-value-here`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/TEMPLATE.md:11]: see comment

File: website/TEMPLATE.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+If the command usage is complex, consider including an example here.
+
+## Configuration
+
+- 📌 `$configValue1`<br />
+  Description of what this configuration value does. The `📌` means that users need to change this value for the command to work (e.g. a channel ID specific to their server).
+
+- `$configValue2`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/TEMPLATE.md:32]: see comment

File: website/docs/birthday/main-cc.md [hunk 13]
```diff
@@ -5,13 +5,27 @@ title: Main CC
 
 This custom command adds birthday functionality to your server, wishing members all the best on their respective date.
 
-**Trigger Type:** `Regex`
+For more information about the birthday system, see [this](overview) page.
 
+## Trigger
+
+**Type:** `Regex`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/main-cc.md:12]: see comment

File: website/docs/birthday/overview.md [hunk 14]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
+Before your birthday custom command is ready to go, you still need to configure a few things. Please read carefully through their description before changing anything.
+
+### List of configurable values
 
-- `$mods`  
+- 📌 `$mods`<br />
   List of role IDs you consider moderator. Separate multiple role IDs by spaces.
-- `$ChannelID`  
+
+- 📌 `$channelID`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:37]: see comment

File: website/docs/birthday/overview.md [hunk 15]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
   The channel that birthday messages should be sent in.
-- `$bdayMsg`  
+
+- `$bdayMsg`<br />
   This variable defines the message that'll be sent on a member's birthday.
-- `$invertedOrder`  
+
+- `$invertedOrder`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:43]: see comment

File: website/docs/birthday/overview.md [hunk 16]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
   Date format to use. If set to to `true`, YAGPDB will use the american notation `mm/dd/yyyy` instead of `dd/mm/yyyy`.
-- `$yearOptional`  
+
+- `$yearOptional`<br />
   Whether the year is optional when configuring birthdays; set to `true` to enable. This will make the year default to `2000`, should a member not give a year upon configuring their birthday.
-- `$kickUnderAge`  
+
+- `$kickUnderAge`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:49]: see comment

File: website/docs/birthday/overview.md [hunk 17]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
-  Syntax: `delbirthday <User:Metion>`  
-  Use: Delete the set birthday of the mentioned user. Can only be used by users with at least one role in `$mods`.
+:::tip
 
-After you've configured all those variables, save again.
+All commands can be used with `bday` or `birthday`, such as `-getbday` or `-getbirthday`.
+
+:::
+
+- `mybirthday`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:71]: see comment

File: website/docs/birthday/overview.md [hunk 18]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
+  **Syntax:** `mybirthday dd/mm/yyyy` or `mybirthday mm/dd/yyyy`  
+  **Use:** Set your birthday to the given date. Note that if `$invertedOrder` is true, you'd have to use the latter version, as it now uses american date notation.
+
+- `startbirthday`<br />
+  **Syntax:** `starbirthday [duration]`  
+  **Use:** Start announcing birthdays at the time of executing this command. Provide an optional duration to fine-tune the timing.
 
-| ✅ Your birthday system is now ready to use! |
-| -------------------------------------------- |
+- `stopbirthday`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:79]: see comment

File: website/docs/birthday/overview.md [hunk 19]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
-After you've configured all those variables, save again.
+All commands can be used with `bday` or `birthday`, such as `-getbday` or `-getbirthday`.
+
+:::
+
+- `mybirthday`<br />
+  **Syntax:** `mybirthday dd/mm/yyyy` or `mybirthday mm/dd/yyyy`  
+  **Use:** Set your birthday to the given date. Note that if `$invertedOrder` is true, you'd have to use the latter version, as it now uses american date notation.
+
+- `startbirthday`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:75]: see comment

File: website/docs/birthday/overview.md [hunk 20]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
+
+- `$yearOptional`<br />
   Whether the year is optional when configuring birthdays; set to `true` to enable. This will make the year default to `2000`, should a member not give a year upon configuring their birthday.
-- `$kickUnderAge`  
+
+- `$kickUnderAge`<br />
   Whether to kick users younger than 13 years old; set to `true` to enable.
-- `$banUnderAge`  
+
+- `$banUnderAge`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:52]: see comment

File: website/docs/birthday/overview.md [hunk 21]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
+
+- `$bdayMsg`<br />
   This variable defines the message that'll be sent on a member's birthday.
-- `$invertedOrder`  
+
+- `$invertedOrder`<br />
   Date format to use. If set to to `true`, YAGPDB will use the american notation `mm/dd/yyyy` instead of `dd/mm/yyyy`.
-- `$yearOptional`  
+
+- `$yearOptional`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:46]: see comment

File: website/docs/birthday/overview.md [hunk 22]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
-- `$mods`  
+- 📌 `$mods`<br />
   List of role IDs you consider moderator. Separate multiple role IDs by spaces.
-- `$ChannelID`  
+
+- 📌 `$channelID`<br />
   The channel that birthday messages should be sent in.
-- `$bdayMsg`  
+
+- `$bdayMsg`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:40]: see comment

File: website/docs/birthday/overview.md [hunk 23]
```diff
@@ -5,104 +5,112 @@ title: Overview
...
 
 ## Configuration
 
-Before your birthday custom command is ready to go, you still need to configure a few things. Please read carefully through their decription before changing anything.
+Before your birthday custom command is ready to go, you still need to configure a few things. Please read carefully through their description before changing anything.
+
+### List of configurable values
 
-- `$mods`  
+- 📌 `$mods`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/birthday/overview.md:34]: see comment

File: website/docs/afk/main-cc.md [hunk 24]
```diff
@@ -1,100 +1,32 @@
...
 ---
 
 This command allows users to set an AFK message with optional duration.
 
-**Trigger Type:** `Regex`
+For more information about the AFK system, please see [this](overview) page.
 
+## Trigger
+
+**Type:** `Regex`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/afk/main-cc.md:12]: see comment

File: website/TEMPLATE.md [hunk 25]
```diff
@@ -0,0 +1,43 @@
...
+
+```
+example command usage
+```
+
+If the command usage is complex, consider including an example here.
+
+## Configuration
+
+- 📌 `$configValue1`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/TEMPLATE.md:29]: see comment

File: website/docs/code-snippets/string2time.md [hunk 26]
```diff
@@ -3,148 +3,54 @@ sidebar_position: 9
...
+1. `dd/mm/yyyy` or `dd.mm.yyyy` or `dd-mm-yyyy` or `dd,mm,yyyy`
+2. String format with year mentioned with 4 digits and both short and long month names supported. Date components (i.e day , month , year) need not be present together. eg: `12 Feb 11:50 am`, `2020` are supported.
+3. `today` and `tomorrow` is supported.
 
-**TimeZone:** By default timezone is UTC. If user has timezone set using `setz` command, timezone adjustment is also possible. UTC time is parsed if explicitly specified UTC in this case.
+**TimeZone:**<br />
+By default timezone is UTC. If user has timezone set using `setz` command, timezone adjustment is also possible. UTC time is parsed if explicitly specified UTC in this case.
 
-**Time:** Time is mentioned as `hh:mm:ss` or `hh:mm` or `hh`. May or may not be followed by AM or PM.
+**Time:**<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/code-snippets/string2time.md:21]: see comment

File: website/docs/code-snippets/string2time.md [hunk 27]
```diff
@@ -3,148 +3,54 @@ sidebar_position: 9
...
 
-- Date: Format 1: `dd/mm/yyyy` or `dd.mm.yyyy` or `dd-mm-yyyy` or `dd,mm,yyyy`
-- Format 2: String format with year mentioned with 4 digits and both short and long month names supported. Date components (i.e day , month , year) need not be present together. eg: `12 Feb 11:50 am` , `2020` is supported.
-- Format 3: `Today` and `tomorrow` is supported.
+1. `dd/mm/yyyy` or `dd.mm.yyyy` or `dd-mm-yyyy` or `dd,mm,yyyy`
+2. String format with year mentioned with 4 digits and both short and long month names supported. Date components (i.e day , month , year) need not be present together. eg: `12 Feb 11:50 am`, `2020` are supported.
+3. `today` and `tomorrow` is supported.
 
-**TimeZone:** By default timezone is UTC. If user has timezone set using `setz` command, timezone adjustment is also possible. UTC time is parsed if explicitly specified UTC in this case.
+**TimeZone:**<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/code-snippets/string2time.md:18]: see comment

File: website/docs/fun/animal.md [hunk 28]
```diff
@@ -1,62 +1,26 @@
...
 title: Animal Image Generator
 ---
 
-This CC generates random images of different animals.
+This command generates random images of different animals.
 
-**Trigger Type:** `Command`
+## Trigger
 
+**Type:** `Command`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/fun/animal.md:10]: see comment

**Comment by l-zeuch** [comment]
re: newlines
just kinda bothers me and my system doesn't like it : / 
I can run it through my system and save all files, that will insert trailing newlines. also GitHub kinda complains about missing trailing newline.

re: Tags
So basically JS poopi?

re: mdx
the more i know :ye:

**Comment by jo3-l** [comment]
newlines: OK, will do.
tags: yes, JS poopi.
mdx: 👍

**Comment by LemmeCry** [comment]
since y'all spamming the logs, I'd like to contribute with this message.

File: website/docs/fun/guess-the-number.md [hunk 29]
```diff
@@ -1,105 +1,46 @@
...
-	{{ end }}
-{{ end }}
-{{ if $embed }}
-	{{ sendMessage nil (cembed $embed) }}
-{{ end }}
+This command runs a "guess the number" game where users send numbers between 1 and 100. The winner wins credits!
+
+## Trigger
+
+**Type:** `Regex`<br />
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply website/docs/fun/guess-the-number.md:10]: and you would be quite correct 😉

**Comment by l-zeuch** [comment]
> since y'all spamming the logs, I'd like to contribute with this message.

perhaps we should make a CoC that says "no unnecessary comments" :ye:

File: website/docs/utilities/edit.md (hunk 8 above)
Comment (jo3-l) [reply website/docs/utilities/edit.md:39]: Don't really like the "However, ...:" construct tbh. Will keep as is unless you have a big objection to this.

File: website/docs/suggestion/main-cc.md (hunk 6 above)
Comment (jo3-l) [reply website/docs/suggestion/main-cc.md:6]: Will rewrite.

File: website/docs/utilities/edit.md (hunk 8 above)
Comment (l-zeuch) [reply website/docs/utilities/edit.md:39]: no objection, but we should definitely include the `-force` flag for the second example (as it is about that flag) to show usage.

File: website/docs/suggestion/main-cc.md (hunk 6 above)
Comment (l-zeuch) [reply website/docs/suggestion/main-cc.md:6]: epico

File: website/docs/utilities/edit.md (hunk 8 above)
Comment (jo3-l) [reply website/docs/utilities/edit.md:39]: Missed that completely 🤦. Will add `-force` to the example.

Comment (l-zeuch) [reply website/docs/utilities/edit.md:39]: remember, children: too much force pushing is not good for you

**Comment by jo3-l** [comment]
This is almost finished 🎉 Still have to update the following:

* README.md (shorten it, point people to the website)
* Contributing guide (needs updates)
* Introduction & guide to adding CCs on the website

...but otherwise we're done here!

**Comment by jo3-l** [comment]
This is now done and ready to review.

**Review by jo3-l — COMMENTED** [review]
I do feel that it's become quite a bit harder to contribute because of the need for adding pages to the website (based on my experience just writing the contribution guide, heh), but perhaps I'm being pessimistic. Will have to wait for feedback from existing contributors though.

File: website/docs/adding-ccs.md [hunk 30]
```diff
@@ -3,54 +3,80 @@ sidebar_position: 2
 title: Adding Custom Commands
 ---
 
-This assumes you know how to create a custom command. If you aren't too familiar with that or just need a refresher, [this article](https://learn.yagpdb.xyz/the-custom-command-interface) explains it well.
+This collection of custom commands is separated into different categories.
+For example, fun commands can be found [here](fun/overview), while the giveaway system is located [here](giveaway/overview).
 
-The **yagpdb-cc** repository is separated into groups / categories. Most categories have standalone custom commands, meaning you can add one, two, or even all of them without issues. There are, however, some "systems" which require you to add all commands in that section. This is detailed in the individual `Read Me`s of that section.
+Generally speaking, there are two kinds of commands. These are as follows:
```
Comment (l-zeuch) [inline website/docs/adding-ccs.md:9]:
```suggestion
Generally speaking, there are two kinds of custom commands. These are as follows:
```
:thinking:

File: website/docs/introduction.md [hunk 31]
```diff
@@ -0,0 +1,40 @@
...
+- **Official:** This collection of commands is [maintained by a group of YAGPDB staff members](#contributors) and has official support.
+- **Well tested:** We review all commands added and many commands have widespread usage, meaning that it's unlikely - [though not impossible](#disclaimer) - for there to be serious bugs in them.
+- **Well documented:** We take pride in our high quality of documentation. It should be _easy_ for someone to figure out how to use any command here given that they are willing to read and have common sense.
+
+Now that you know a little about the project, we suggest that you take a look around to see if there's something that you like. If you're not sure of how to add custom commands from here, please refer to [this page](adding-ccs).
+
+## Disclaimer
+
+The YAGPDB developer, staff, and/or support are not responsible for any issues with or caused by these custom commands. Furthermore, though we thoroughly review all commands added, they are not guaranteed to be working; use them at your own risk.
+
```
Comment (l-zeuch) [inline website/docs/introduction.md:19]: see the first comment regarding disclaimer

**Review by l-zeuch — DISMISSED** [review]
[insert some witty summarizing comment here]

File: README.md [hunk 32]
```diff
@@ -2,101 +2,48 @@
...
-## Need help?
-
-If you are having difficulties with one of the custom commands here, please get in touch via the [YAGPDB Support Server](https://discord.gg/5uVyq2E), where we can help you further. You can also open an issue on the issue tracker if that's what you prefer.
-
-## Contributors
-
-**yagpdb-cc** is currently maintained by [@jo3-l](https://github.com/jo3-l), [@Satty9361](https://github.com/Satty9361), [@l-zeuch](https://github.com/l-zeuch), [@Pedro-Pessoa](https://github.com/Pedro-Pessoa) and [@DaviiD1337](https://github.com/DaviiD1337).<br>
+# Stop!
+
+Are you looking to browse/possibly add some custom commands from this repository? If so, we highly suggest doing so on our [website](https://yagpdb-cc.github.io) instead. This GitHub repository's main purpose is to act as a place to develop custom commands, rather than as a place for users to view them.
```
[the remark this answers is not in the document]
Comment (l-zeuch) [reply README.md:15]: yo what the fuck why is this diff so huge

File: CONTRIBUTING.md [hunk 33]
```diff
@@ -1,85 +1,157 @@
...
 ```
 
-We named our configuration variable `ROLE_ID`, in UPPER_SNAKE_CASE, as it makes it clear that it is a constant. Note that this is not required, just recommended.
+Now, you're done with changes to your code. It's time to write summon your inner writer and some documentation for your command.
+
+## Write some documentation for the website
+
+> **Note:** We know this part can be a bit confusing, so feel free to open an issue or just skip this part and ask for help when you make a Pull Request to the repository.
+
+One of the cool things about this repository is that we have a website! What's pretty cool about it is that you can write normal Markdown - the file format of what you're reading right now - and have it come out looking like a consistently styled document.
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply CONTRIBUTING.md:118]: i agree, but this needs a rewrite anyways (the contrib guide). don't want to do it in this PR though, not worth blocking this huge change for some rewriting.

File: README.md (hunk 32 above)
[the remark this answers is not in the document]
Comment (jo3-l) [reply README.md:15]: it's redundant, yes, but I'd like to get the point across that we highly recommend users to use the website. 2x the text, 2x harder to miss :D

File: src/code_snippets/LICENSE.md [hunk 34]
```diff
@@ -23,4 +23,4 @@ OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
 ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
 
-For more information, please refer to http://unlicense.org/
+For more information, please refer to &lt;<http://unlicense.org/>&gt;
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply src/code_snippets/LICENSE.md:26]: 💢

File: README.md [hunk 35]
```diff
@@ -2,101 +2,48 @@
...
+A curated collection of over custom commands for the [YAGPDB Discord Bot](https://yagpdb.xyz) to enhance your server. It is:
+
+- **Extensive:** There are over 60 custom commands (and we're adding more regularly!), meaning that it's likely that you'll be able to find what you're looking for.
+- **Official:** This collection of commands is [maintained by a group of YAGPDB staff members](#contributors) and has official support.
+- **Well tested:** We review all commands added and many commands have widespread usage, meaning that it's unlikely - [though not impossible](#disclaimer) - for there to be serious bugs in them.
+- **Well documented:** We take pride in our high quality of documentation. It should be _easy_ for someone to figure out how to use any command here given that they are willing to read and have common sense.
+
+# Disclaimer
+
+The YAGPDB developer, staff, and/or support are not responsible for any issues with or caused by these custom commands. Furthermore, though we thoroughly review all commands added, they are not guaranteed to be working; use them at your own risk.
```
[the remark this answers is not in the document]
Comment (jo3-l) [reply README.md:28]: honestly don't think it's necessary, none of the scripts are controversial (hopefully???)

**Comment by jo3-l** [comment]
Still got one thing to do before we :shipit:, gonna see if I can get the website pages to reorder automatically using some JS magic

File: README.md (hunk 32 above)
[the remark this answers is not in the document]
Comment (l-zeuch) [reply README.md]: fair enough :mmlol:

File: README.md (hunk 35 above)
[the remark this answers is not in the document]
Comment (l-zeuch) [reply README.md:28]: tbh i agree, but nowadays it's better safe than sorry - we can always add it later  should issues like this occur.

i say we keep it in mind for possible future #cancelculture. no need to block the PR for that.

**Comment by jo3-l** [comment]
Think this is ready to go, last commit just changed around some metadata (no content changed, despite the giant diff) : )

**Pull request merged**